### PR TITLE
Cancel reservation

### PR DESCRIPTION
**Pull request by Arribas12**
When you click a reservation and then you click the button cancel reservation it is deleted from the database

### CODE REVIEW

File: src/main/java/es/deusto/spq/client/gui/views/reservations/ReservationDetailView.java [hunk 1]
```diff
@@ -88,7 +91,14 @@ public void refresh() {
 
         // Cancel button always gets updated (locale)
         cancelButton.setText(getViewManager().getClient().getLocaleManager().getMessage("reservation.detail.button.cancel"));
+        cancelButton.addActionListener(new ActionListener() {
 
+        	@Override
+        	public void actionPerformed(ActionEvent e) {
+        		getViewManager().getClient().getController().deleteReservation(reservationDTO.getId());
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/client/gui/views/reservations/ReservationDetailView.java:108]: The window should be closed after cancelling

File: src/main/java/es/deusto/spq/server/remote/HotelManager.java [hunk 2]
```diff
@@ -415,6 +415,14 @@ public ReservationDTO createReservation(String reservationId, String userId, Str
 	}
 
 	@Override
+	public boolean deleteReservation(String reservationId) throws RemoteException {
+		boolean state = reservationDao.deleteReservationByID(reservationId);
+		if(state == true) {
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/server/remote/HotelManager.java]: `return state` will do the same thing

File: src/main/resources/localization_es_ES.properties [hunk 3]
```diff
@@ -1,4 +1,190 @@
...
+test.parameters.1 = Si a�ades 1 y 2, obtienes {0, number, integer}
+test.parameters.2 = Hola, {0}. Tienes {1} pago pendiente.
+
+app.name = HotelMS
+
+menu.about.title = Sobre HotelMS
+menu.quit = Cerrar
+menu.about.body = <html>\
+  <b>HotelMS</b><br><br>Creado por:<br>\
+  Íñigo Apellániz (iapellaniz@opendeusto.es)<br>\
```
Comment (iapellaniz) [inline src/main/resources/localization_es_ES.properties]: :'(

**Review by iapellaniz — CHANGES_REQUESTED** [review]

**Review by Egoes98 — CHANGES_REQUESTED** [review]
Only one of the windoes is closed, could yo close the twoo of them?? I mean that it closes the one where you see the info and the other  keep open so you still see the information until you close the View my reservations and open it again.

**Comment by Egoes98** [comment]
> Only one of the windoes is closed, could yo close the twoo of them?? I mean that it closes the one where you see the info and the other keep open so you still see the information until you close the View my reservations and open it again.

I am doing it myself, I have nearly finished.

**Comment by Egoes98** [comment]
> > Only one of the windoes is closed, could yo close the twoo of them?? I mean that it closes the one where you see the info and the other keep open so you still see the information until you close the View my reservations and open it again.
> 
> I am doing it myself, I have nearly finisheI have tried doing a setter but is trykier than i thougt. I dont know if I would be able. Look at how to do it please.

**Comment by Egoes98** [comment]
Okay I have done it. Now both are closing.

**Review by iapellaniz — APPROVED** [review]

**Review by Egoes98 — APPROVED** [review]

**Pull request merged**